### PR TITLE
🐎 Pony

### PR DESCRIPTION
**Pull request by TimoFreiberg**
Thanks to Steve, Alex, Constantin and the remote participants

### CODE REVIEW

File: pony/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Pony
+
+## How to
+
+* Install [ponyc](https://github.com/ponylang/ponyc)
+* Run `ponyc`
```
Comment (bitterblue) [inline pony/README.md]: This step fails for me with

```
Building builtin -> /usr/lib/pony/0.28.0/packages/builtin
Building . -> /home/axel/CompetenceGroups/AQE/hello-world-in-90-languages/pony
Error:
http: couldn't locate this path
Error:
/home/axel/CompetenceGroups/AQE/hello-world-in-90-languages/pony/hello-world.pony:1:1: can't load package 'http'
use "http"
^
```

**Review by bitterblue — CHANGES_REQUESTED** [review]
I guess the dependency is declared in `bundle.json`, but how do I resolve it?

File: pony/README.md (hunk 1 above)
Comment (TimoFreiberg) [reply pony/README.md]: Yeah, woops. ponyc can't handle dependencies - i'll update the README to recommend https://github.com/ponylang/pony-stable

Comment (TimoFreiberg) [reply pony/README.md]: There's no date header yet... woops

**Review by bitterblue — APPROVED** [review]

**Pull request merged**